unit_scaling/_modules.py

```diff
@@ -4,43 +4,75 @@
 
 from __future__ import annotations  # required for docs to alias type annotations
 
-from typing import Any, Optional
+from typing import Any, List, Optional, Tuple, Union
 
 import einops
+import torch
 from torch import Tensor, nn
 
 from . import functional as U
 from ._internal_utils import generate__all__
+from .core.functional import ResidualScalingFn, transformer_residual_scaling_rule
 from .docs import (
     binary_constraint_docstring,
     format_docstring,
     inherit_docstring,
+    mult_docstring,
     variadic_constraint_docstring,
 )
+from .parameter import MupType, Parameter, has_parameter_data
 
 
 @inherit_docstring(
     short_description="Applies a **unit-scaled** Gaussian Error Linear Units function:",
-    add_args=[binary_constraint_docstring],
+    add_args=[mult_docstring(), binary_constraint_docstring],
     unsupported_args=["approximate"],
```

Comment:
Not critical, but is there any reason this can't support approximate? Surely the approximation is good enough that the `mult` can be the same whether using `erf` or `tanh` implementation.

Reply:
I agree, should be close enough; I'll remove this check.